ihmc-perception/src/main/java/us/ihmc/perception/realsense/RealsenseDevice.java

```diff
@@ -3,7 +3,21 @@
 import org.bytedeco.javacpp.BytePointer;
 import org.bytedeco.javacpp.FloatPointer;
 import org.bytedeco.librealsense2.global.realsense2;
-import org.bytedeco.librealsense2.*;
```

Comment:
Probably better with the *

Reply:
Isn't this just the formatter? Shouldn't we stick to those guidlines so its universal?

Reply:
I thought that was the guideline but apparently it's 20 lines before it collapses